oddbird/sassdoc-theme-herman
# [Bug] Herman preview styles overriden by custom styles

![](https://source.unsplash.com/featured/?cute,animal&CHANGE_ME)




## Description

- Fixes styles for code examples being overriden by site custom styles. 
- Limits the amount of Herman color custom properties generated for color previews. 




## Steps to test/reproduce
- To see a live version of the conflicts, see the color previews on [this OddSite PR](https://deploy-preview-276--oddleventy.netlify.app/styleguide/config-color) using the latest version of Herman.  
- the font sizes are too big and the some of the Herman styles for code examples are being overrwritten by custom styles from OddSite. 
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._


## Show me

Color previews now only contain two generated custom properties: `--herman-brand-blue` and `--herman-brand-pink`:

![bild](https://user-images.githubusercontent.com/41588129/202012707-1d918b85-0238-4e94-9935-54903d5b3cdf.png)

Previously we were generating custom properties for all Herman colors:

Old:

![bild](https://user-images.githubusercontent.com/41588129/202013083-a10c21e4-f898-4571-99bd-f126aaf93eb1.png)

_Provide screenshots/animated gifs/videos if necessary._
